### PR TITLE
Remove syntax highlighting rule that breaks codemirror

### PR DESCRIPTION
**Pull request by minestarks**
This causes JupyterLab to throw an error when opening .qs documents. 

They were way too many backslashes in the regex... the CodeMirror parser didn't like that, failed parsing, which brought down the whole editor for some reason.

### CODE REVIEW

File: jupyterlab/src/index.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ function registerQSharpLanguage(
     },
     {
       token: "operatorKeyword",
-      regex: String.raw`(=)|(!)|(<)|(>)|(\\+)|(-)|(\\*)|(\\/)|(\\^)|(%)|(\\|)|(\\&\\&\\&)|(\\~\\~\\~)|(\\.\\.\\.)|(\\.\\.)|(\\?)`,
```
Comment (billti) [inline jupyterlab/src/index.ts:71]: So.... was this never working/parsing right?

Shouldn't `(\\/)` still leave one slash? (ie.g. `(\/)` instead of `(/)`?

Comment (minestarks) [reply jupyterlab/src/index.ts:71]: No, it was never working.

`/` is not a special character in regex, they just need to be escaped when using the `/regex/` syntax. We're using a string here so it should be fine.

https://javascript.info/regexp-escaping#a-slash

So intuitive right?

Comment (minestarks) [reply jupyterlab/src/index.ts:71]: Just to be clear: this line was never working, but it was only relatively recently added. I didn't mean to imply syntax highlighting was broken from the start.

Comment (billti) [reply jupyterlab/src/index.ts:71]: OK. I've no idea how CodeMirror works, so as long as it's not getting eval'ed into a RegEx literal or something funky like that, the / should be fine (I think it's benign to still escape though). Thanks!

**Review by billti — APPROVED** [review]

**Pull request merged**